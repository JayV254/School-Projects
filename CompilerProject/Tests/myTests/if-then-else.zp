int main() {
    int a;
    puts("This function will square an input number...input a number that is less than 20 below\n");
    a = getnum();

    if(a < 20) then {
        puts("The square of your number is: ");
	putn(a*a);
    }
    else {
        puts("Sorry your number is greater than 20");
	puts("\n");
    }
    
    return 0;
}

