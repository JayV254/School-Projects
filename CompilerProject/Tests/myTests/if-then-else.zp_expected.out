This function will square an input number...input a number that is less than 20 below
input-> 4 
The square of your number is:     16 
